boltops-tools/terraspace
# fix completion_script

This is a 🐞 bug fix.






## Summary

Fixes autocomplete. 

Note, autocomplete is ok. Feel like it can be improved, performance is not as fast as would like it.

## Context

Fixes #63

## How to Test

Add to your `~/.bash_profile`

    eval $(terraspace completion_script)

Start a new terminal and type:

    terraspace n[TAB]
    terraspace new [TAB]
    terraspace new project [TAB]

It should autocomplete, should see something like this:

    $ terraspace new project --
    --bundle          --examples        --help            --plugin          --quiet           --test-structure
    --config          --force           --lang            --plugin-gem      --test            
    $ terraspace new project --
    
## Version Changes

Patch
